WordPress/jetpack_metadata/PlayStoreStrings.po

```diff
@@ -11,6 +11,26 @@ msgstr ""
 "Project-Id-Version: Jetpack - Apps - Android - Release Notes\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_190"
+msgid ""
+"19.0:\n"
+"- Manage your dashboard with a card-based feed, including Posts.\n"
+"- Upload up to 5-minute videos on free plans.\n"
+"- See more user interface text in your language, including Jetpack/Layout Grid blocks.\n"
+"- Keep text formatting after hitting backspace.\n"
+"- Block titles are now center-aligned in the inserter menu.\n"
+msgstr ""
+
+#. translators: Shorter Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_short_190"
+msgid ""
+"19.0:\n"
+"- Card-based dashboard feed, including Posts.\n"
+"- Maximum of 5-minute video uploads on free plans.\n"
+"- More user interface translation, including Jetpack/Layout Grid blocks.\n"
+"- Consistent text formatting after hitting backspace.\n"
+msgstr ""
+
```

Comment:
@mokagio FYI, I had to _manually_ add the entry for `release_note_short_190` in both WP & JP's `PlayStoreStrings.po`, because our toolkit failed to do so. See a343637914a844edd95a58e1787c6bd645cb2b63.

As a reminder, technically speaking this is because our current logic in the release-toolkit duplicates the `release_note_*` blocks from the previous release when generating the new ones, but 18.9 did not have a short version of release notes. This is not the first time that this happened and that we got caught in this issue due to poor logic in our toolkit.

Since you're precisely working on fixing that `.po` file generation as part of our L10n project, so I thought I'd let you know here ti provide you with a concrete example of conditions for when that happens 🙃

Reply:
Thank you for pointing this out. It's a good reminder 👍 

> Since you're precisely working on fixing that `.po` file generation as part of our L10n project...

I opened https://github.com/wordpress-mobile/release-toolkit/pull/326 just a few hours ago to add a test that documents the existing behavior, but I did not include the case for the short notes. I'll either update that or followup with a new one.

Now that I think about it, it might be cool to add some "expected to fail" tests (I'm sure RSpec has some kind of option to do that) with the _desired_ behavior, to document the quirks such as the one you outlined here.